command/shell.go

```diff
@@ -34,6 +37,21 @@ func (command *ShellCommand) Execute([]string) error {
 	}
 	fmt.Printf("Type 'exit' to exit. Use ctrl-c to abort running commands.\n\n")
```

Comment:
I think we should move this line down below m.FilterInstances, since we are still validating the configuration at this point and users don't have a prompt to type 'exit' in etc.